src/Card/CardImageCap.jsx

```diff
@@ -47,24 +48,29 @@ const CardImageCap = React.forwardRef(({
 
   const handleSrcFallback = (event, altSrc) => {
     const { currentTarget } = event;
-    if (currentTarget.src !== altSrc) {
+
+    if (!altSrc || currentTarget.src.endsWith(altSrc)) {
+      currentTarget.style.display = 'none';
```

Comment:
If we're now using the `show` class to control the image visibility, is setting the `display` style to "none" here redundant? Should we instead rely solely on the `show` state variable rather than having 2 potentially competing approaches to hiding the element?

Reply:
I wonder if this function could possible get simplified to something similar to this:

```jsx
const handleSrcFallback = (event, altSrc, imageKey) => {
  const { currentTarget } = event;
  if (!altSrc || currentTarget.src === altSrc) {
    setShow(s => ({ ...s, [imageKey]: false }));
    return;
  }
  currentTarget.src = altSrc;
};
```

where `imageKey` is `imageCap` or `logoCap` depending on the caller of `handleSrcFallback`. 

[aside] I wonder if having separate state variables (one per image) would simplify this suggestion at all, though.

Reply:
I agree, two different approaches are not what we need

Reply:
> [aside] I wonder if having separate state variables (one per image) would simplify this suggestion at all, though.

I'm pretty sure it would, in this case using composite state does not give any benefit I think, just makes code a bit harder to understand

@PKulkoRaccoonGang let's introduce 2 states (`showImageCap`, `showLogoCap`) instead of one, this should simplify code